vm/vm.go

```diff
@@ -215,33 +216,37 @@ func (v *vm) Execute(txns []core.Transaction, declaredClasses []core.Class, bloc
 	return context.actualFees, context.traces, nil
 }
 
-func marshalTxnsAndDeclaredClasses(txns []core.Transaction, declaredClasses []core.Class) (json.RawMessage, json.RawMessage, error) {
+func marshalTxnsAndDeclaredClasses(txns []core.Transaction, declaredClasses []core.Class) (json.RawMessage, json.RawMessage, int, error) {
 	txnJSONs := []json.RawMessage{}
-	for _, txn := range txns {
-		txnJSON, err := marshalTxn(txn)
+	var queryBitBitmap int
```

Comment:
Neat idea, but it fails when `len(txns) >= (sizeof(int) * 8)`

Reply:
Oh my goodness. Sleep deprived coding gone wrong :sweat_smile: 
Thanks!